frontend/vue/components/MiniComposer/Gate.vue

```diff
@@ -68,7 +67,7 @@ export default class Gate extends Vue.with(Props) {
 .gate {
```

Comment:
should we add a `hover` cursor, here? the same way we have one for the Eigenvector widget?

https://user-images.githubusercontent.com/6276074/173419592-94b44f2e-8d31-4844-b1b8-49164b406e99.mov